pkg/apis/ceph.rook.io/v1/scc.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	_ "embed"
+	"fmt"
+
+	secv1 "github.com/openshift/api/security/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewSecurityContextConstraints returns a new SecurityContextConstraints for Rook-Ceph to run on
```

Comment:
@leseb same SCC is needed for CSI also? do we have any plan for it?

Reply:
After more discussion, we decided not to include CSI since Rook is not the right component to expose those. I'm planning on discussing this during the CSI weekly.

Reply:
Sounds good. Thank you